SQL/0000-00-00-schema.sql

```diff
@@ -762,6 +802,33 @@ CREATE TABLE `mri_protocol_checks` (
     FOREIGN KEY (`Scan_type`) REFERENCES `mri_scan_type` (`ID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `mri_protocol_checks_group` (
+    `MriProtocolChecksGroupID`   INT(2) UNSIGNED NOT NULL AUTO_INCREMENT,
+    `Name`                       VARCHAR(255)    NOT NULL,
+    PRIMARY KEY (`MriProtocolChecksGroupID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+INSERT INTO `mri_protocol_checks_group` VALUES(1, 'Default MRI protocol checks group');
+
+CREATE TABLE `mri_protocol_checks_group_rel` (
+    `MriProtocolChecksGroupID` INT(2) UNSIGNED  NOT NULL,
+    `MriProtocolChecksID`      INT(11)          NOT NULL,
+    CONSTRAINT `FK_mri_protocol_checks_group_rel_1` FOREIGN KEY (`MriProtocolChecksGroupID`) REFERENCES `mri_protocol_checks_group` (`MriProtocolChecksGroupID`),
+    CONSTRAINT `FK_mri_protocol_checks_group_rel_2` FOREIGN KEY (`MriProtocolChecksID`)      REFERENCES `mri_protocol_checks` (`ID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+CREATE TABLE `mri_protocol_checks_group_target` (
+     `MriProtocolChecksGroupID` INT(2) UNSIGNED  NOT NULL,
+     `ProjectID`                INT(2)           DEFAULT NULL,
+     `SubprojectID`             INT(10) UNSIGNED DEFAULT NULL,
+     `Visit_label`              VARCHAR(255)     DEFAULT NULL,
+     CONSTRAINT `FK_mri_protocol_checks_group_target_1` FOREIGN KEY (`MriProtocolChecksGroupID`) REFERENCES `mri_protocol_checks_group` (`MriProtocolChecksGroupID`),
+     CONSTRAINT `FK_mri_protocol_checks_group_target_2` FOREIGN KEY (`ProjectID`)                REFERENCES `Project` (`ProjectID`),
+     CONSTRAINT `FK_mri_protocol_checks_group_target_3` FOREIGN KEY (`SubprojectID`)             REFERENCES `subproject` (`SubprojectID`)
+) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
+
+INSERT INTO `mri_protocol_checks_group_target` VALUES(1, NULL, NULL, NULL);
```

Comment:
see comment on the insert statement at line 811

Reply:
ok